src/requirements.txt

```diff
@@ -2,6 +2,7 @@
 aiohttp==4.0.0a1
 async-timeout==3.0.1
 attrs==19.3.0
+boto3==1.14.9
```

Comment:
You'll want to add this dependency to the `Pipfile`. This `requirements.txt` is generated.

Reply:
Thank you for the quick response!

I made an attempt to resolve this. The `Pipfile` now looks as I would expect. The `Pipfile.lock` has a boatload of changes, as does the `requirements.txt`, generated from `pipenv`. Here are the steps I took (I'm new to `pipenv`):

```
pipenv shell
pipenv install boto3
pipenv lock -r > src/requirements.txt
```
Assuming the resulting `Pipfile.lock` and `src/requirements.txt` are _not_ correct, tips on how to get them properly updated would be most appreciated.